misk-redis/src/main/kotlin/misk/redis/RedisConfig.kt

```diff
@@ -2,8 +2,26 @@ package misk.redis
 
 import misk.config.Config
 
-data class RedisConfig(
-  val host_name: String,
-  val port: Int,
-  val auth_password: String
-) : Config
+/**
+ * The top level field is the name of the replication group.
+ * See: https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/WhatIs.Terms.html
```

Comment:
these are aws-specific configs/comments. Do we want that to live in the misk-redis package?

Reply:
Since we are abstracting away the aws-specific details with this layer I think you're right. Removing.